ion/src/device/n0110/drivers/board.cpp

```diff
@@ -64,7 +62,7 @@ void initMPU() {
   /* This is needed for interfacing with the LCD
    * We define the whole FMC memory bank 1 as strongly ordered, non-executable
    * and not accessible. We define the FMC command and data addresses as
-   * writeable non-cacheable, non-buffereable and non shareable. */
+   * writeable non-cachable, non-buffereable and non shareable. */
```

Comment:
Why are you editing it ?
```suggestion
   * writeable non-cacheable, non-buffereable and non shareable. */
```